src/components/Dashboard/CourtStats.js

```diff
@@ -1,20 +1,152 @@
 import React from 'react'
-import { Box, Distribution } from '@aragon/ui'
+import { Box, GU, textStyle, useTheme } from '@aragon/ui'
+import useCourtStats from '../../hooks/useCourtStats'
+import { formatUnits } from '../../lib/math-utils'
+import {
+  useANJBalanceToUsd,
+  useTokenBalanceToUsd,
+} from '../../hooks/useTokenBalanceToUsd'
+import Loading from './Loading'
 
-const CourtStats = () => {
+const splitAmount = amount => {
```

Comment:
This looks like it might've been taken from Finance, or another app who didn't get the memo on the aragonDS upgrade.

Wondering if we should:

1. Make this an actual component (e.g. `SplitAmount`)
2. Update the css to use `GU` and `textStyle()`